test_drupal_quicksprint.sh

```diff
@@ -15,12 +15,15 @@ tar -C /tmp -zxf ~/tmp/drupal_sprint_package.no_docker.$(cat ./.quicksprint_rele
 # Run the standard install_ddev.sh, asserting we already have docker set up.
 cd /tmp/drupal_sprint_package && echo y | ./install_ddev.sh
 
+
 # Remove any existing sprints, then do start_sprint.sh
-cd ~/Sites/sprint && chmod -R u+w sprint-* && rm -rf sprint-* echo y | ./start_sprint.sh
+cd ~/Sites/sprint
+find ~/Sites/sprint -type d -name "sprint-2*" -exec chmod -R u+w '{}' \; -exec rm -rf '{}' \;
```

Comment:
I always have trouble using `xargs`, which is more efficient than `-exec`.

Reply:
xargs is nice, `find ~/Sites/sprint -type d -maxdepth 1 -name "sprint-2*" | xargs rm -r` ; that doesn't solve the fact that you want to do both the chmod and the rm.